Dockerfile.erb

```diff
@@ -1,20 +1,10 @@
-FROM debian:jessie
+FROM debian:stable
 
 RUN groupadd -r bitcoin && useradd -r -m -g bitcoin bitcoin
 
 RUN set -ex \
 	&& apt-get update \
-	&& apt-get install -y --no-install-recommends ca-certificates wget \
-	&& rm -rf /var/lib/apt/lists/*
-
-# grab gosu for easy step-down from root
-RUN set -ex \
-	&& gpg --keyserver keyserver.ubuntu.com --recv-keys B42F6819007F00F88E364FD4036A9C25BF357DD4 \
-	&& wget -qO /usr/local/bin/gosu "https://github.com/tianon/gosu/releases/download/1.7/gosu-$(dpkg --print-architecture)" \
-	&& wget -qO /usr/local/bin/gosu.asc "https://github.com/tianon/gosu/releases/download/1.7/gosu-$(dpkg --print-architecture).asc" \
-	&& gpg --verify /usr/local/bin/gosu.asc \
-	&& rm /usr/local/bin/gosu.asc \
-	&& chmod +x /usr/local/bin/gosu
+	&& apt-get install -qy --no-install-recommends ca-certificates dirmngr gosu gpg wget
```

Comment:
Does `-q` actually do anything here? in my tests it appears to show the exact same output.

Awesome that `gosu` is included in debian now, that is much cleaner.